examples/scheduled_email.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import resend
+
+if not os.environ["RESEND_API_KEY"]:
+    raise EnvironmentError("RESEND_API_KEY is missing")
+
+params: resend.Emails.SendParams = {
+    "from": "onboarding@resend.dev",
+    "to": ["delivered@resend.dev"],
+    "subject": "hi",
+    "html": "<strong>hello, scheduled email!</strong>",
+    "scheduled_at": "2024-09-05T11:52:01.858Z",
+}
+
+# Throws a resend.exceptions.ValidationError
+# when scheduled_at is not in ISO 8601 format
```

Comment:
Let's add (as a comment) an example on how to create a ISO 8601 format date string. Something like:
```
# Throws a resend.exceptions.ValidationError
# when scheduled_at is not in ISO 8601 format.
# Here is an example on how to create a date in the ISO 8601 format:
# from datetime import datetime
# datetime.now().isoformat()
```